data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/S3ObjectPluginMetrics.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.Timer;
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+
+public class S3ObjectPluginMetrics {
```

Comment:
I think this. Nice!

Reply:
This class should have a unit test.

Reply:
Thanks David for your review, Changes are incorporated and available for review.

Reply:
Thanks David for your review, Changes are incorporated and available for review.